DMD2.h

```diff
@@ -168,10 +168,10 @@ class DMDFrame
 
   void swapBuffers(DMDFrame &other);
 
-  const byte width; // in pixels
-  const byte height; // in pixels
+  const uint16_t width; // in pixels
+  const uint16_t height; // in pixels
  protected:
-  volatile uint8_t *bitmap;
+  volatile uint8_t *bitmap;uint16_t
```

Comment:
I don't think you meant to change this line. Can you please fix this up? Thanks.

Reply:
Yes. Those should have been typed by mistake. I have edited the code.
Thank you for pointing that out.
```suggestion
  volatile uint8_t *bitmap;
```